README.md

```diff
@@ -129,13 +127,14 @@ Click exit icon at the top right of the widget and you'll see the empty form.
 # What happened under the hood
 ## Token exchange flow
 
-First we initiate the widget using your `public_key`. 
+First we make a call to our API to request a bridge token from Citadel and use this bridge token to initiate the Citadel Bridge.
 
 ```
+        const bridgeToken = await apiRequests.getBridgeToken()
         const bridge = CitadelBridge.init({
           clientName: 'Citadel Quickstart',
-          companyMappingId: null,
-          key: '{{ public_key }}',
+          companyMappingId: bridgeToken.settings.company_mapping_id,
```

Comment:
We should not need this parameter. @rjriel pls check if this works, and report to eng-ux if not.